app/helpers/api_helper.rb

```diff
@@ -1,29 +1,18 @@
 module ApiHelper
-  # Shape for a paginated response
-  def paginated(json, pagy)
+  def paginated(json)
     json.data do
       yield
     end
 
-    json.links do
-      scaffold_url = pagy_metadata(pagy, absolute: true)[:scaffold_url]
-      url_for_page = ->(page) do
-        next nil if page.nil?
-        scaffold_url.gsub("__pagy_page__", page.to_s)
+    unless @page.last?
+      json.links do
+        json.next url_for(page: @page.next_param)
```

Comment:
It looks like this is a breaking change because we're losing the keys `first`, `prev`, `last`, and `self`.

Could we implement those links so that we don't have a breaking change?